app/components/profile_picture/status.tsx

```diff
@@ -39,8 +39,8 @@ const Status = ({author, statusSize, statusStyle, theme}: Props) => {
         statusStyle,
         {borderRadius: statusSize / 2},
     ]), [statusStyle]);
-
-    if (author?.status && !author.isBot) {
+    const isBot = author && (('isBot' in author) ? author.isBot : author.is_bot);
```

Comment:
not really part of the PR, but in app/components/profile_picture/index.tsx we have a call to get the user status, I wonder if we should skip it if the user is a bot.